tests/funcalign/test_srm.py

```diff
@@ -62,6 +63,9 @@ def test_can_instantiate():
 
     # Check that runs with 2 subject
     s.fit(X)
+    sr_v0_4 = np.load('sr_v0_4.npy')
+    diff_v0_4 = np.linalg.norm(sr_v0_4 - s.s_)
+    assert(diff_v0_4 < 1e-5)
```

Comment:
Shouldn't the results be element-wise equal, i.e., `np.allclose(sr_v0_4, s.s_)`, maybe with an `atol` value?